apps/ledger-live-mobile/src/components/ServicesWidget/Protect/AddPaymentProtectState.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useCallback } from "react";
+import { useTranslation } from "react-i18next";
+import { Tag } from "@ledgerhq/native-ui";
+import { Linking } from "react-native";
+import Button from "../../Button";
+
+function AddPaymentProtectState({
+  params,
+}: {
+  params: Record<string, string>;
+}) {
+  const { t } = useTranslation();
+
+  const { addNowURI, viewDetailsURI } = params || {};
+
+  const onAddNow = useCallback(() => {
+    Linking.canOpenURL(addNowURI).then(() => Linking.openURL(addNowURI));
+  }, [addNowURI]);
+
+  const onViewDetails = useCallback(() => {
+    Linking.canOpenURL(viewDetailsURI).then(() =>
+      Linking.openURL(viewDetailsURI),
+    );
+  }, [viewDetailsURI]);
+
+  return (
+    <>
+      <Button type="main" outline={false} onPress={onAddNow} mt={8} mb={6}>
+        {t(`servicesWidget.protect.status.addPayment.actions.addNow`)}
+      </Button>
+      <Button type="default" outline={false} onPress={onViewDetails}>
+        {t(`servicesWidget.protect.status.addPayment.actions.viewDetails`)}
+      </Button>
+    </>
+  );
+}
+
+const StateTag = () => {
+  const { t } = useTranslation();
+
+  return (
+    <Tag
+      color="hsla(36, 94%, 56%, 1)"
```

Comment:
It's not in the theme ?
I'm afraid it will not work in dark or light mode :eagle:

Reply:
It was a design choice they wanted a specific color :shrug: